.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ references:
   container_config_node: &container_config_node
     working_directory: ~/project
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:14
```

Comment:
This _might_ still need `-browsers` on the end. @i-like-robots would you know?

Reply:
I don't think so, it's got a very straightforward test setup 🤔